Code/GraphMol/DetermineBonds/DetermineBonds.h

```diff
@@ -0,0 +1,23 @@
+//
+//  Copyright (C) 2022 Sreya Gogineni and other RDKit contributors
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include <RDGeneral/export.h>
+#pragma once
```

Comment:
We generally use include guards in header files, not `#pragma once` (which isn't part of any actual C++ standard).

Please use the usual RDKit pattern:
```
#ifndef RD_DETERMINEBONDS_H
#define RD_DETERMINEBONDS_H
```
(look at other RDKit header files for examples)